src/apps/frontend/pages/tasks/task-header.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+
+import { Button, Flex, HeadingLarge } from '../../components';
+import { AsyncError } from '../../types';
+import { ButtonSize } from '../../types/button';
+import { TaskOperationType } from '../../types/task';
+
+import TaskModal from './task-modal';
+import useTaskForm from './tasks-form.hook';
+
+interface TaskHeaderProps {
+  onError?: (error: AsyncError) => void;
+}
+
+const TaskHeader: React.FC<TaskHeaderProps> = ({ onError }) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const onSuccess = () => {
+    toast.success('Task has been added successfully');
+    setIsModalOpen(false);
+  };
+
+  const { addTaskFormik, setFormikFieldValue } = useTaskForm({
+    onError,
+    onSuccess,
+  });
+
+  return (
+    <div className="rounded-sm  border border-stroke bg-white p-3 shadow-default">
+      <div className="flex flex-col items-start justify-between gap-5 sm:flex-row sm:items-center ">
+        <Flex>
```

Comment:
In order to make our website responsive, we need to use flex because in mobile mode, our user interface will appear as a column.

Reply:
Here it just contains only one item in this flex, so it will act as a simple div. And also the responsiveness is added to its parent, so I think adding the flex to this heading doesn't make any difference.